run_addb_batch.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -ex
+
+addb_dir="/var/log/addb_res/"
+s3bpath="../s3bench/s3bench"
+
+s3bpath=$(realpath -e "$s3bpath")
+mkdir -p "$addb_dir"
+
+test_plan=$(cat <<-EOF
+[
+{"clients": 1, "size": "100b", "reads": 10000},
```

Comment:
Will this run just two s3 bench tests?
We may have to run multiple tests (more than these two tests)
Would developer need to modify the script (i.e add more json rows here) to add more tests?

Reply:
these tests are just an example. you should edit it write the tests you need to run. you can add any number tests